### DISCUSSION TITLE
Add support for message export option to sync latest messages

### DISCUSSION THREAD

**Discussion by carlos-moreira** [d99kris/nchat#289] [Ideas]
From what I understand, to export the message cache using the command line option:
`-x, --export <DIR> `
you will always get the cache from the last time nchat was opened and synced the messages, right?

So, to get the latest messages, I'm always forced to first open nchat, let the syncing happens, and only after all messages are synced and cached, I would run the --export command to get the "latest state".

Is there a way to make the --export command to always force check if there are new messages? And if so, cache them first, and only after that export the files. This way there won't be the need to open nchat first.

**Comment by d99kris** [comment]
Hi @carlos-moreira - your understanding is correct, and there is no support for performing sync automatically as part of export today. I'll convert this thread to a feature request, and maybe it can be added some day. Can I check if you're mainly using WhatsApp, Telegram or both?

**Reply by carlos-moreira** [reply]
I’m using for WhatsApp currently.
